### PR TITLE
Issue #180 check progress bug

### LINKED ISSUE #180: Check progress bug
Check progress prints 11%  instead of 100% with status 1100.

### PR DESCRIPTION
**Pull request by zarooba01**
resolves #180

### CODE REVIEW

File: gui/ajax.php [hunk 1]
```diff
@@ -262,7 +262,12 @@ function ajaxCheckProgress(){
...
 	foreach ($active_sims as $row) {
 		$iteration = $row['iteration'];
-		$status = substr($row['status'],-4,2);
+		if($row['status']=='1100'){
+			$status = substr($row['status'],-3,3);
+		}
+		else{
+			$status = substr($row['status'],-2);
+		}
 		if ($status == "00") { $status="100";}
```
Comment (kowalskiw) [inline gui/ajax.php:271]: Can this line be reached at all since we catch 1100 status before?

File: gui/halt.php [hunk 2]
```diff
@@ -107,11 +107,15 @@ function check_stat($r) {
...
-			$statuses['in progress'] += 1;
-			echo "<td>".($element['status']-1000)."%</td><td>Job in progress</td></tr>"; // Add a line break after each inner array
-		}else{
+        if ($element['status'] > 1000 & $element['status']<1100 ){
+            $sum += 1;
+            $statuses['in progress'] += 1;
+            echo "<td>".($element['status']-1000)."%</td><td>Job in progress</td></tr>"; // Add a line break after each inner array
+        }
+        elseif($element['status'] == 1100){
+                echo "<td>".($element['status']-1000)."%</td><td>Job finished</td></tr>";
```
Comment (kowalskiw) [inline gui/halt.php:116]: Job of status 100% is still in progress I suppose - meta file is saved and results are exported to DB. Statuses 0-99 stand for finished eventually.
I suggest to restore those changes - to be discussed. Unless there are bugs in halt.php regarding issue at hand?

**Review by kowalskiw — CHANGES_REQUESTED** [review]
@zarooba01 - just two comments to be addressed before merge.

File: gui/ajax.php (hunk 1 above)
Comment (zarooba01) [reply gui/ajax.php:271]: You're right.

File: gui/halt.php (hunk 2 above)
Comment (zarooba01) [reply gui/halt.php:116]: Understandable. 
No, everything works well.

**Review by kowalskiw — APPROVED** [review]

**Pull request merged**